connect_your_app_to_IRIS/technical_details/app_onboarding.md

```diff
@@ -347,9 +520,9 @@ AttendanceInformation object:
 | `attendTo` | Attend to | true |
 | `additionalInformation` | Additional attendance information | false | For example table or area 
 
-## Test your implementation
+## 5 Test your implementation
 
-To test your implementation, visit [https://iris.staging.iris-gateway.de](https://iris.staging.iris-gateway.de) 
+To test your implementation, visit [https://iris.staging.iris-gateway.de:9443](https://iris.staging.iris-gateway.de) 
```

Comment:
Without port 9443 is okay, we have switched.

Reply:
Yeah, link was already changed but link text left unchanged. In this case a link text is useless, so I changed to just the URL.